bindings/README.md

```diff
@@ -154,11 +154,8 @@ output_match_mode: substring
 expected_stdout_lines: 
   - "You're up and running! Both Dapr and your app logs will appear here."
   - "== APP == {'data': {'orderId': 1}, 'operation': 'create'}"
-  - "== APP == <Response [204]>"
   - "== APP == {'data': {'orderId': 2}, 'operation': 'create'}"
-  - "== APP == <Response [204]>"
   - "== APP == {'data': {'orderId': 3}, 'operation': 'create'}"
-  - "== APP == <Response [204]>"
```

Comment:
Can we keep at least the last one? I feel at some point, the calls must succeed so we don't hide an issue.

Reply:
> Can we keep at least the last one? I feel at some point, the calls must succeed so we don't hide an issue.

done!